src/docs/truffle/reference/configuration.md

```diff
@@ -131,6 +131,24 @@ For each network, you can specify `host` / `port`, `url`, or `provider`, but not
 - https://sandbox.truffleteams.com/yoursandboxid
 - wss://sandbox.truffleteams.com/yoursandboxid
 
+#### Special (managed) networks: `test` and `develop`
+
+The networks `test` and `develop` are special, in that they can be used to specify the settings for the managed Ganache instance started by `truffle test` and `truffle develop`, respectively.  These options are simply included like any other network options.  The supported options are:
+
+* `network_id`: The network ID to use.  Defaults to 5777.
+* `total_accounts`: The number of unlocked accounts Ganache should provide.  Defaults to 10.  Alias: `accounts`.
+* `default_balance_ether`: The default amount of ether to seed unlockd accounts with, specified in ether.  Defaults to 100 ether.  Alias: `defaultEtherBalance`.
```

Comment:
Thanks for documenting this, @haltman-at. Its already merged but just found a small typo in the word `unlockd`.

Reply:
Oops, thanks!